exchange/bidder_test.go

```diff
@@ -704,8 +704,8 @@ func TestMultiCurrencies(t *testing.T) {
 			1,
 			currencyConverter.Rates(),
 			&adapters.ExtraRequestInfo{},
-			true,
-			true,
+			nil,
```

Comment:
Same for all of these tests. A NilSigner would be self-documenting and easier for the reader to understand.

Reply:
Yes, replaced in all occurrences